HardwareReport/index.html

```diff
@@ -59,6 +59,9 @@ <h3 hidden>Compasses</h3>
 <h3 hidden>Barometers</h3>
 <p id="BARO"></p>
 
+<h3 hidden>GPS</h3>
+<p id="GPS"></p>
```

Comment:
Needs to be added to the list of stuff that is cleared when a new log is loaded.

Reply:
Added it to the reset() function body to clear the children before loading a new log,